cypress/e2e/login.cy.js

```diff
@@ -1,11 +1,32 @@
-/// <reference types="cypress" />
+/// <reference types = "cypress" />
 
 describe('Login page', () => {
   beforeEach(() => {
-    
+    cy.visit('https://the-internet.herokuapp.com/login');
   });
 
-  it('', () => {
-    
+  it('Should be able to login with valid credentials', () => {
```

Comment:
```suggestion
  it('should allow to login with valid credentials', () => {
```
sounds better if we want the `it` name to continue the `describe` name